addons/website_slides/static/src/js/slides_upload.js

```diff
@@ -76,6 +76,7 @@ var SlideUploadDialog = Dialog.extend({
                 self.$modal.find('.modal-dialog').addClass('modal-lg');
             }
             self._bindSelect2Dropdown();
+            self._fetchSections();
```

Comment:
Would move this to willStart then, as now we need to pre-select the "last section".

That way sections are properly loaded as thie widget loads and we don't need to make manual calls to "_fetchSections".